src/meta/src/stream/stream_manager.rs

```diff
@@ -105,6 +110,136 @@ where
         })
     }
 
+    #[async_recursion]
```

Comment:
Better not using `#[async_recursion]` here, try to fill `chain_upstream_table_node_actors` firstly outside.